src/Resources/skeleton/web/.htaccess

```diff
@@ -2,14 +2,12 @@
     RewriteEngine On
 
     <IfModule mod_headers.c>
-        # Automatically add caching for 1 year on assets residing
-        # in /assets as well as /bundles which are all assets that contain a hash
-        # in their respective filenames or are called using some ?version parameter
-        # for cache busting.
+        # Assets in /assets and /bundles either contain a hash in their filename
+        # or are called with a ?version suffix, therefore cache them for 1 year.
         RewriteRule ^(assets|bundles)/ - [ENV=CONTAO_ASSETS:true]
         Header set Cache-Control "max-age=31536000" env=CONTAO_ASSETS
 
-        # Allow CORS on any tinymce4 skins
+        # Allow CORS on The Contao TinyMCE skin.
```

Comment:
> The

Shouldn't that be lower case?

Reply:
Fixed in 97e3365534cebb2cf35bec9d4aa39733e2abd5bb.